CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 ## Unreleased
 
+## 0.7.3
```

Comment:
Since this is a breaking change, I think we should increment the version to `0.8.0`.